sql/mob_droplist.sql

```diff
@@ -21929,9 +21929,8 @@ INSERT INTO `mob_droplist` VALUES (2639,0,0,1000,1031,30); -- Gusgen Chest Key (
 INSERT INTO `mob_droplist` VALUES (2639,2,0,1000,880,0);   -- Bone Chip (Steal)
 
 -- ZoneID: 192 - Wendigo Blm
--- ZoneID: 198 - Wendigo Blm
-INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,880,210);    -- Bone Chip (21.0%)
-INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,544,90);     -- Pile Of Answer Sheets (9.0%)
+INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,880,@VCOMMON);    -- Bone Chip (Very Common, 24%)
+INSERT INTO `mob_droplist` VALUES (2640,0,0,1000,544,@RARE);     -- Pile Of Answer Sheets (Rare, 5%)
```

Comment:
Comment needs shifted to align with comment of previous row.

Reply:
you know 90% the file doesn't align right? I forget who started that. I'd rather we just have a a name field and shift all of them into it (saying the rate after is redundant, esp after aI made the `@variables`).

DSP shot down such suggestions back in the day saying "you can use a join query" as if we're going to review by running queries locally. They ain't here - just need 2 devs to approve. I'm one.